internal/admission/admission.go

```diff
@@ -65,9 +71,20 @@ func (r *Router) InjectDecoder(d *admission.Decoder) error {
 }
 
 func (r *Router) Handle(ctx context.Context, req admission.Request) admission.Response {
-	if h, ok := r.handlers[req.RequestKind.Kind]; ok {
-		return h.Handle(ctx, req)
+	h, ok := r.handlers[req.RequestKind.Kind]
+	if !ok {
+		return admission.Errored(http.StatusBadRequest, fmt.Errorf("resource %s not implemented", req.RequestKind.Kind))
+	}
+
+	cfg, err := r.limitRanger.LimitRangeConfig(req.Namespace)
+	if err != nil {
+		return admission.Errored(http.StatusBadRequest, fmt.Errorf("failed to retrieve limit range information from namespace %s: %s", req.Namespace, err.Error()))
+	}
+
+	if cfg == nil {
+		return admission.Allowed(fmt.Sprintf("No container limit range in namespace: %s", req.Namespace))
 	}
 
-	return admission.Errored(http.StatusBadRequest, fmt.Errorf("resource %s not implemented", req.RequestKind.Kind))
+	ctx = context.WithValue(ctx, LimitRangeContextTypeMemory, cfg)
```

Comment:
Sorry late to the party but the only downside using context.WithValue is that Handling code needs to do dynamic type checking:

```
	config, ok := ctx.Value(admission.LimitRangeContextTypeMemory).(*limitrange.Config)
	if !ok {
		err := fmt.Errorf("failed to get Config from context")
		klog.Log.Error(err, "context does not contain %s", admission.LimitRangeContextTypeMemory)
		return admissionruntime.Errored(http.StatusInternalServerError, err)
	}
```

No preference from me but thought I'd bring it up